pages/api/extension/trigger.ts

```diff
@@ -48,12 +49,19 @@ async function triggerDPU(url: string, userEmail: string) {
 		console.error(`[triggerDPU] Unable to get user for alias ${userEmail}, error = ${err}`);
 		throw new Error("Unable to get user from db");
 	});
-	if (users?.length == 0) {
+	if (!users || users?.length == 0) {
```

Comment:
What if the `users` object is non-null, but doesn't contain the `length` field? Will that still be handled?


```suggestion
	if (users?.length ?? 0 === 0) {
```
Wouldn't this be better?

Reply:
I tried this and users was resolving to `DbUser [] | undefined` after this if block. The undefined didn't go away until I put `!users`